autocertkit/testbase.py

```diff
@@ -242,10 +242,20 @@ def generate_static_net_conf(self):
                         log.debug("Create static config for %s (%s)" %
                                   (iface, vlan))
                         key_name = "%s_%s" % (iface, vlan)
-                        assert(key_name not in res.keys())
+                        assert key_name not in res.keys(
+                        ), "Duplicate static IP addressing specified for %s (%s)" % (iface, vlan)
                         res[key_name] = sm
                         log.debug("Added static conf for '%s'" % key_name)
 
+        mgmt = get_pool_management_device(self.session)
+        if 'static_management' in netconf and mgmt not in netconf:
+            log.debug("Create static config for mgmt device %s" % mgmt)
+            key_name = "%s_0" % (mgmt)
+            assert key_name not in res.keys(
+            ), "'static_management' should only be specified when management device(%s) is not being tested for certification." % (iface)
+            res[key_name] = StaticIPManager(netconf['static_management'])
```

Comment:
Not sure about this... static manager is used for issue and retrieve IP(s) used during ACK. If it is management, it should be fixed. I am not entirely sure whether it is appropriate to add it to manager.